arch/arm/src/armv7-a/arm.h

```diff
@@ -19,10 +19,10 @@
  ****************************************************************************/
 
 /* References:
- *  "Cortex-A5� MPCore, Technical Reference Manual", Revision: r0p1,
- *   Copyright � 2010 ARM. All rights reserved. ARM DDI 0434B (ID101810)
- *  "ARM� Architecture Reference Manual, ARMv7-A and ARMv7-R edition",
- *   Copyright � 1996-1998, 2000, 2004-2012 ARM. All rights reserved.
+ *  "Cortex-A5™ MPCore, Technical Reference Manual", Revision: r0p1,
```

Comment:
is this conversion right?

Reply:
I think so? What do you thing is the problem?

Reply:
it looks good on my smart phone, but wrong on my PC. The locale config may have some problem in my PC.